etc/dnf/aliases.d/command-abbreviations.conf

```diff
@@ -0,0 +1,19 @@
+[aliases]
+dup = distro-sync
+dsync = distro-sync
+dist-upgrade = distro-sync
+dg = downgrade
+in = install
+mc = makecache
+ref = makecache
+refresh = makecache
+rei = reinstall
+ri = reinstall
+rm = remove
+se = search
+sh = shell
+up = upgrade
+um-m = upgrade-minimal
```

Comment:
Wasn't this `up-m`?

Reply:
Almost, it was ``u-m``. Thanks for catching that.